_episodes/09-branches.md

```diff
@@ -87,60 +69,46 @@ one development line (branch) and this branch is called "master":
 
 ![]({{ site.baseurl }}/img/gitink/git-branch-1.svg)
 
-```shell
-$ git log --stat
-
-commit 7f3582dfbad6539cfa60f5b21bfad41d1b58a618
-Author: Radovan Bast <bast@users.noreply.github.com>
-Date:   Fri May 5 12:49:45 2017 +0200
-
-    do not forget to enjoy
-
- instructions.txt | 1 +
- 1 file changed, 1 insertion(+)
-
-commit 64441c1934def7d91ff0b66af0795749d5f1954a
-Author: Radovan Bast <bast@users.noreply.github.com>
-Date:   Fri May 5 12:49:35 2017 +0200
+We have defined the `git graph` alias earlier using:
 
-    add onion
-
- ingredients.txt | 1 +
- 1 file changed, 1 insertion(+)
+```shell
+$ git config --global alias.graph "log --all --graph --decorate --oneline --abbrev-commit"
```

Comment:
is `--abbrev-commit` ever needed these days?

Reply:
I will check and probably remove.

Reply:
Yes, we need that flag. Otherwise we get the full hashes.

Reply:
This is strange.  In my man page:

       --oneline
           This is a shorthand for "--pretty=oneline --abbrev-commit" used
           together.

... going back at least to 1.8.3.1

Reply:
Sorry, my bad -  I made a mistake when testing your question. You are right. I will remove it.